tilelang/language/v2/builder.py

```
@@ -958,7 +1054,7 @@ def substitute_buffer(buf):
     )
 
 
-def prim_func(func: Callable[_P, _T] = None, *, lazy_jit=False) -> PrimFunc[_P, _T] | LazyJITFunc[_P, _T]:
+def prim_func(func: Callable[_P, _T] = None, *, lazy_jit: bool = False) -> PrimFunc[_P, _T] | LazyJITFunc[_P, _T]:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)